flycheck.el

```diff
@@ -6872,6 +6873,22 @@ See URL `https://github.com/zaach/jsonlint'."
      (and (buffer-file-name)
           (string= (file-name-extension (buffer-file-name)) "json")))))
 
+(flycheck-define-checker json-pythonjson
+  "A JSON syntax checker using Python json.tool module.
+
+See URL `https://docs.python.org/3.5/library/json.html#command-line-interface'."
+  :command ("python" "-m" "json.tool" source-inplace null-device)
```

Comment:
Why do you use `source-inplace` here?  Is that really necessary?  Can't we use `source` instead?

I presume `null-device` is for the (pretty-printed?) output?   If so, please add a little comment that explains it, so that I don't need to look up documentation if I need to understand this syntax checker in half a year from now or so :relaxed: